src/modules/launcher/PowerLauncher.Telemetry/Events/LauncherHideEvent.cs

```diff
@@ -1,9 +1,14 @@
 ﻿using System.Diagnostics.Tracing;
+using Microsoft.PowerToys.Telemetry;
+using Microsoft.PowerToys.Telemetry.Events;
 
 namespace Microsoft.PowerLauncher.Telemetry
 {
     [EventData]
-    public class LauncherHideEvent
+    public class LauncherHideEvent : EventBase, IEvent
     {
+        public PartA_PrivTags PartA_PrivTags => PartA_PrivTags.ProductAndServiceUsage;
+
+        PartA_PrivTags IEvent.PartA_PrivTags => throw new System.NotImplementedException();

```

Comment:
[clarification] Why is this required when the `PartA_PrivTags ` is specified above?

Reply:
Whoops I'll get rid of that.